concert/base.py

```diff
@@ -662,9 +667,13 @@ def lower(self):
 
     @lower.setter
     def lower(self, value):
+        if value is None:
+            self._lower = None
+            return
         self._check_limit(value)
-        if value >= self._upper:
-            raise ValueError('Lower limit must be lower than upper')
+        if self._upper is not None:
+            if value >= self._upper:
```

Comment:
You could simply write ` if self._upper is not None and value >= self._upper:`

Reply:
Yes, you are right. At one point I had a problem where this did not work, but now it works and I will change it.